app/pb_kits/playbook/pb_date/docs/_date_timezone.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= pb_rails("caption", props: { text: "East Coast (Default)"}) %>
```

Comment:
```suggestion
<%= pb_rails("caption", props: { text: "East Coast (Default)" }) %>
```
Minor nitpick...you blank spaced after the left  `{` but not after the right. This goes for all uses of the caption and body kits throughout the rest of this file.

Reply:
I shall add those extra blank spaces. (Taylor Swift would approve.)